index.js

```diff
@@ -1,9 +1,11 @@
-var syntaxHighlight = require( "./lib/highlight" );
+"use strict";
+
+const syntaxHighlight = require( "./lib/highlight" );
 
 exports.syntaxHighlight = syntaxHighlight;
 
 exports.postPreprocessors = {
-	_default: function( post, fileName, callback ) {
+	_default( post, _fileName, callback ) {
```

Comment:
Did you add this underscore on purpose?

Reply:
Yes, our ESLint config requires all variables & function parameters to be used inside of the function body. For parameters it's not possible in 100% of cases and then prefixing with an underscore is an opt-out.

Reply:
We should probably use [after-used](https://eslint.org/docs/rules/no-unused-vars#args-after-used)

Reply:
We had a discussion about that some time ago and we decided to use `all` as there are more cases where we'd leave internal unused parameters than in APIs where they have to be there. This applies especially to AMD module definitions where we've been constantly missing unused dependencies.

We had to prefix some parameters with `_` in Core because of this rule but there weren't too many of them,